code/game/area/areas.dm

```diff
@@ -522,8 +593,9 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 	if(outdoors)
 		return FALSE
 	areasize = 0
-	for(var/turf/open/T in get_contained_turfs())
-		areasize++
+	for (var/list/zlevel_turfs as anything in get_zlevel_turf_lists())
+		for(var/turf/open/thisvarisunused in zlevel_turfs)
+			areasize++
```

Comment:
please stay consistent in your use of space after for (at least for the file/code adjacent to each other)

Reply:
also couldn't this just be += length(zlevel_turfs)

Reply:
I'll never write a non-spaced version and contributing.md prohibits me from changing existing control statements between spaced and non-spaced

Reply:
contributing.md can suck my nuts. I prefer non spaced, write most my stuff like that, but if you're going through changing procs should be consistent in the proc. how you do that is up to you tho